src/Application/Staff/Commands/UpsertStaffMemberProfile/UpsertStaffMemberProfileCommand.cs

```diff
@@ -55,6 +55,13 @@ public async Task<StaffMemberDto> Handle(UpsertStaffMemberProfileCommand request
         // Add if doesn't exist
         if (staffMemberEntity == null)
         {
+            // Check if email already exists
+            var emailConflict = await _context.StaffMembers
```

Comment:
Can we add this, so we have the comment in SQL?

```suggestion
            var emailConflict = await _context.StaffMembers
                .TagWithContext("CheckDuplicateEmail")
```